classes/class-wc-connect-functions.php

```diff
@@ -175,7 +175,7 @@ public static function backup_existing_tax_rates() {
 				__( 'Shipping', 'woocommerce' ) . ',' .
 				__( 'Tax Class', 'woocommerce' ) . "\n";
 
-			echo $header;
+			echo $header; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Can we convert this into a HTML-only PHP file that we require instead of doing variable assignment then `echo`?

Reply:
I prefer to have a follow-up to clean this up. The reason is we may be reworking some of these classes very soon.